flit.ini

```diff
@@ -4,10 +4,15 @@ author=Jupyter Development Team
 author-email=jupyter@googlegroups.com
 home-page=https://github.com/jupyter/testpath
 description-file=README.rst
-dev-requires = pathlib2; python_version == "2.7" or python_version == "3.3"
+dev-requires = pathlib2; python_version == "2.7"
+python_requires=">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*"
 classifiers = Intended Audience :: Developers
     License :: OSI Approved :: MIT License
     Programming Language :: Python
     Programming Language :: Python :: 2
+    Programming Language :: Python :: 2.7
     Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.4
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.6
```

Comment:
Please don't add these; I don't like using classifiers for this.

Reply:
Removed.